axon-server-connector/src/test/java/org/axonframework/axonserver/connector/AxonServerConfigurationTest.java

```diff
@@ -0,0 +1,62 @@
+package org.axonframework.axonserver.connector;
+
+import org.junit.jupiter.api.Test;
+
+import static org.axonframework.axonserver.connector.AxonServerConfiguration.builder;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
```

Comment:
Another nit: Would you mind adding a short piece of class level javadoc describing what's exactly being tested in this class?